<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtSkillModelGenerate</class>
 <widget class="QDialog" name="QtSkillModelGenerate">
  <property name="windowModality">
   <enum>Qt::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1087</width>
    <height>975</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>110</y>
     <width>1071</width>
     <height>861</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gLayoutPreview"/>
  </widget>
  <widget class="QPushButton" name="btn_generate">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>生成预览</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_export">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>70</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>导出配置</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>44</y>
     <width>36</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>关键字</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>74</y>
     <width>36</width>
     <height>12</height>
    </rect>
   </property>
   <property name="text">
    <string>替换为</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="group_skinid">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>20</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>皮肤ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLineEdit" name="edit_src_skinid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_dst_skinid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_skinname">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>皮肤名</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLineEdit" name="edit_src_skinname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_dst_skinname">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_skillid">
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>技能ID</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLineEdit" name="edit_src_skillid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_dst_skillid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="group_roleid">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>450</x>
     <y>20</y>
     <width>160</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>RoleId</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLineEdit" name="edit_src_roleid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="edit_dst_roleid">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
